decidim-proposals/app/models/decidim/proposals/collaborative_draft.rb

```diff
@@ -28,7 +28,7 @@ class CollaborativeDraft < Proposals::ApplicationRecord
                class_name: "Decidim::User",
                foreign_key: :decidim_user_id
 
-      geocoded_by :address, http_headers: ->(collaborative_draft) { { "Referer" => collaborative_draft.component.organization.host } }
+      geocoded_by :address
```

Comment:
Yeah, this option in the `geocoded_by` did nothing in the previous implementation if you check the geocoder documentation:
https://github.com/alexreisner/geocoder#advanced-model-configuration

> Supported parameters: `:lookup`, `:ip_lookup`, `:language`, and `:params`.

You could remove the same from current core code and it would have zero effect on the functionality of `geocoded_by`.

Now with the new API, all these go through the Decidim API thanks to [this module](https://github.com/decidim/decidim/blob/1eba56e6439b2598131b3d15919ca85193e51d99/decidim-core/lib/decidim/geocodable.rb), so you no longer have to particularly configure the gem for the geocoded attributes.